clusterloader2/testing/load/config.yaml

```diff
@@ -0,0 +1,120 @@
+name: Load
+automanagedNamespaces: 3
+tuningSets:
+- name: Uniform100qps
```

Comment:
I think you should change these QPSs to averageQPS instead of uniform to simulate load test accurately.